components/org.wso2.carbon.identity.tenant.resource.manager/pom.xml

```diff
@@ -1,4 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change the year to 2020?

Reply:
Since this code was added in 2019 license was added wrt that